src/plugins/muc-views/templates/add-muc.js

```diff
@@ -21,6 +22,7 @@ const nickname_input = (o) => {
 export default (o) => {
     const i18n_join = __('Join');
     const i18n_enter = __('Enter a new Groupchat');
+    const i18n_jid_placeholder = __('Enter Server Name');
```

Comment:
There was already a placeholder being used `o.chatroom_placeholder`. I think you should just continue using that instead of creating your own one (and leaving the original one as dead code not being used).

Reply:
Sure, will fix it.